test/e2e/features/basic.feature

```diff
@@ -135,7 +135,7 @@ Feature: Basic test
     Scenario: Monitoring stack check
         Given checking that CRC is running
         And executing "crc oc-env | Invoke-Expression" succeeds
-        And login to the oc cluster succeeds
+        And Set kubeconfig context to "crc-admin" succeeds
```

Comment:
I am wondering how the CI is passing if this step is not defined anymore 🤔

Reply:
@guillaumerose because it was for windows platform so doesn't run on any CI.